cmd/ocitools/validate.go

```diff
@@ -397,6 +398,26 @@ func checkLinuxResources(r rspec.Resources, hostCheck bool) (msgs []string) {
 			msgs = append(msgs, fmt.Sprintf("Minimum memory limit should be larger than memory reservation"))
 		}
 	}
+	if r.Network != nil && hostCheck {
+		var exist bool
+		interfaces, err := net.Interfaces()
+		if err != nil {
+			msgs = append(msgs, err.Error())
+			return
+		}
+		for _, prio := range r.Network.Priorities {
+			exist = false
+			for _, ni := range interfaces {
+				if prio.Name == ni.Name {
+					exist = true
+					break
+				}
+			}
+			if !exist {
+				msgs = append(msgs, fmt.Sprintf("Interface %s does not exist on host", prio.Name))
```

Comment:
[`linux.network`][1] is a strange setting, because with cgroup setup happening during creation (also before post-create hooks, or whatever we end up calling them, opencontainers/runtime-spec#532) there's no time for hooks or post-create activity to populate a new network namespace before this cgroup is configured.  Is this setting only intended for containers joining (or staying in) existing network namespaces?  Or can you set priorities for interfaces that don't exist yet, knowing that you'll be creating the interfaces after create finishes up?

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config-linux.md#network